packages/ui/src/Chart/CandleChart.tsx

```diff
@@ -491,12 +491,19 @@ const CandleChart = ({
 
   // Update liquidation range data when it changes
   useEffect(() => {
-    if (!liquidationRange) return
-
     const setSeriesData = (series: ISeriesApi<'Area'> | null, data: any) => {
       if (series) series.setData(data)
     }
 
+    // Clear all series if liquidationRange is undefined or has no data
+    if (!liquidationRange || (!liquidationRange.current && !liquidationRange.new)) {
```

Comment:
It's a bit fucked up how `liquidationRange` itself can be non-null while at the same time all the data inside can be🤔